include/dca/phys/models/general_interaction.hpp

```diff
@@ -118,6 +121,7 @@ std::vector<int> general_interaction<parameters_type>::make_correlated_orbitals(
         for (auto b_j : interacting_orbitals) {
           for (int s_i = 0; s_i < SpinDmn::dmn_size(); ++s_i) {
             for (auto b_i : interacting_orbitals) {
+              // This 1.e-3 seems like a very magic number!
               if (std::abs(H_interaction(b_i, s_i, b_j, s_j, delta_r)) > 1.e-3) {
```

Comment:
Indeed, we need to pick only the non-zero elements. But a check against zero (or something like 10^-10) would do, as no arithmetic is performed on H_interaction